### ISSUE TITLE
New .net content not on XYZ site

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#787]
From Lucy J Parry: 

Not everything has been exported to the new site. (e.g. the Privy Council methods which we did on the old site because we needed to share it, are not on there in their updated form).

**Comment by jesicarson** [comment]
I remember this was going to be an issue as we elected to not build a tool to automatically update the data from .net to .xyz once we did launched. Need to develop a system to track all new edits on .net and manually transfer them to .xyz. Unless something has changed since I’ve been away. Should get on this sooner rather than later. Let me know if I can help!

**Comment by scottofletcher** [comment]
@jesicarson Yes, good idea, perhaps that's what Dethe intended me to file an issue for - I brought Lucy's comment/question up with him before posting this and he said the same thing IE we need a tracking system so we don't miss anything. Same should probably go for other pages as well (help pages, about, etc.)

**Comment by dethe** [comment]
Just to be clear, this issue is for new things on .net since we last exported to .xyz, correct? At first it read to me that not all things in .net _at the time_ had been brought over, which would be my first priority if it were the case.

**Comment by scottofletcher** [comment]
That's correct - this issue relates to everything that has been enter on .net since the last export. I can't remember when that was but there've been quite a few additions and deletions since the last export.

**Comment by dethe** [comment]
Given the (small, I think) amount of new content on .net vs. the amount of development time it would require, how about this instead:

1. Implement admin tools to edit authors and dates
2. Copy new content over manually
3. Profit!!!

**Comment by scottofletcher** [comment]
When was the last time we pushed .net to XYZ? Beginning of March? 

Since March 1st there have been 20 cases, 15 methods/tools, and 8 orgs published on .net
